src/java/org/apache/cassandra/cql3/statements/SelectStatement.java

```diff
@@ -885,9 +885,9 @@ private void maybeWarn(ResultSetBuilder result, QueryOptions options)
         ColumnFamilyStore store = cfs();
         if (store != null)
             store.metric.coordinatorReadSize.update(result.getSize());
-        if (result.shouldWarn(options.getCoordinatorReadSizeWarnThresholdKB()))
+        if (result.shouldWarn(options.getCoordinatorReadSizeWarnThresholdBytes()))
         {
-            String msg = String.format("Read on table %s has exceeded the size warning threshold of %,d kb", table, options.getCoordinatorReadSizeWarnThresholdKB());
+            String msg = String.format("Read on table %s has exceeded the size warning threshold of %,d", table, options.getCoordinatorReadSizeWarnThresholdBytes());
```

Comment:
Would it make sense to replace "kb" w/ "bytes" instead of nothing?

Reply:
Another idea...what if we just used the actual `DataStorageSpec` instances as fields in `DefaultTrackWarnings` and nulls in `DisabledTrackWarnings`? Then we could use the `toString()` here.

Reply:
main reason I didn't do that is due to the fact `maybeFail` is called frequently (each row), so didn't like calling `toBytes` constantly... One option is to have the interface hold both, but feels like overkill; `bytes` I think is a good blend as humans can always convert to a higher form if needed, but know explicitly what the limit is